editor/src/components/inspector/inspector.tsx

```diff
@@ -615,6 +625,7 @@ export const SingleInspectorEntryPoint: React.FunctionComponent<{
   // in order to prevent a big rerender of the inspector
 
   const inspectorModelMemoized = useKeepReferenceEqualityIfPossible(inspectorModel)
+  const targetsMemoized = useKeepReferenceEqualityIfPossible(targets)
```

Comment:
We shouldn't be calling these `*Memoized`. Also it looks like that `FIXME` above is dead.

Reply:
renamed!